src/deprecations.jl

```diff
@@ -1 +1,47 @@
-### v0.8 Deprecations
+### Deprecated while v0.7 was latest
```

Comment:
aren't you deprecating these now in 0.8 lifecycle?

Reply:
They did not gain messages nor move to this file, but were no longer the preferred way once the versions re-using `y` were added. I guess that's all I want to capture. Nothing can be removed until 0.9.